lib/main.js

```diff
@@ -33,7 +33,14 @@ class TypeScriptLanguageClient extends AutoLanguageClient {
   }
 
   deactivate() {
-    return Promise.race([super.deactivate(), this.createTimeoutPromise(2000)])
+    let deactivate = super.deactivate();
+    let cancel = new Promise(function(resolve, reject) {
```

Comment:
Thanks for this.  Can we go with the lambda syntax for these functions (e.g. `(resolve, reject) =>` instead of ```function(resolve, reject)``` to keep it consistent with the rest of the file/project?

Reply:
Sure thing